onko_dicom.py

```diff
@@ -0,0 +1,81 @@
+# pylint: disable=unnecessary-lambda
```

Comment:
I don't like
Ryan Thoughts?

Reply:
.idea directory shouldnt be committed at least

Reply:
also not sure the purpose of this file